package.json

```diff
@@ -160,7 +160,8 @@
       "react/prop-types": "warn",
       "jsx-quotes": "warn",
       "react/jsx-tag-spacing": "warn",
-      "react/no-unused-prop-types": "warn"
+      "react/no-unused-prop-types": "warn",
+      "prefer-const": "warn"
```

Comment:
Can we have a linter config, so that we cannot merge into dev unless there's no linter warning?

Reply:
Change warn to error here. That's the reason we have warn and error :P

Reply:
Why would we ever want to have warnings in dev? 🤔

Reply:
And do lint errors prevent building / starting / testing?

Reply:
They shouldn't. At least I never experienced that it would prevent it.

Reply:
Hmm... so should we make everything an error ? :D

Reply:
If thats your definition of less annoying :P
Let's move this discusison to Slack :D